src/main/java/com/sonyericsson/rebuild/RebuildAction.java

```diff
@@ -297,7 +297,13 @@ public void doConfigSubmit(StaplerRequest req, StaplerResponse rsp) throws Servl
                 }
             }
 
-            List<Action> actions = constructRebuildCause(build, new ParametersAction(values));
+            Set<ParameterValue> mergedValues = ImmutableSortedSet.copyOf(new Comparator<ParameterValue>() {
+                @Override
+                public int compare(ParameterValue o1, ParameterValue o2) {
+                    return o1.getName().compareTo(o2.getName());
+                }
+            }, Iterables.concat(values, paramAction.getParameters()));
```

Comment:
Probably you are right, the action will be always set and it's never null, but, in general, I think it's a good practice to check for null the return value of `getAction`, since it depends on someone adding that action before (and in some cases even in a separate plugin, which is clearly _butterfly effect_ prone :smile: IMO)